netbox/templates/dcim/device.html

```diff
@@ -26,6 +26,12 @@ <h2 class="card-header">{% trans "Device" %}</h2>
                         <th scope="row">{% trans "Location" %}</th>
                         <td>{% nested_tree object.location %}</td>
                     </tr>
+                    {% if vc_members %}
```

Comment:
This should be conditional on the value of `object.virtual_chassis`, rather than `vc_members`.